libraries/AP_AHRS/AP_AHRS.cpp

```diff
@@ -820,6 +820,8 @@ Vector3f AP_AHRS::wind_estimate(void) const
 bool AP_AHRS::airspeed_estimate(float &airspeed_ret) const
 {
     bool ret = false;
+
+#if AP_AIRSPEED_ENABLED && !(APM_BUILD_TYPE(APM_BUILD_ArduCopter) && BOARD_FLASH_SIZE < 1025)
```

Comment:
```suggestion
#if AP_AIRSPEED_ENABLED && !(APM_BUILD_TYPE(APM_BUILD_ArduCopter) && BOARD_FLASH_SIZE <= 1024)
```

Reply:
Fixed all of them. Thanks that is clearer.